package.json

```diff
@@ -38,16 +38,16 @@
   "dependencies": {
     "@babel/core": "^7.11.0",
     "@babel/polyfill": "^7.10.4",
-    "broccoli-funnel": "^2.0.2",
-    "broccoli-merge-trees": "^3.0.2",
+    "broccoli-funnel": "^3.0.3",
+    "broccoli-merge-trees": "^4.2.0",
```

Comment:
I'm pretty sure that these are completely unused at this point, can you confirm and remove?

Reply:
Done